README.md

```diff
@@ -18,7 +18,7 @@ As such, anytime `CreateObject("Scripting.<object>")` is used, this error will a
 ## Installation and Use
 Right click on the project in the projectect explorer then choose import file. Import any necessary files into your project. From there, call the objects to use them (i.e. `Dim dict As New Dictionary`).
 
-### Documentation
+### Similar Documentation
```

Comment:
This is done because there are minor changes between the two dictionaries. These changes should be added to the documentation.